### DISCUSSION TITLE
可否增加手工刷新服务器文件的功能

### DISCUSSION THREAD

**Discussion by AspirantH** [AlistGo/alist#1812] [Q&A] [answered]
本地传了一个文件到123pan，隔了好久alist列表中还未显示出来，请问能否增加一个手工刷新列表的功能

**Comment by xhofe** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/36558727/192222107-5c96ad5f-1ee5-426c-8eea-ae75edce463e.png)

**Comment by AspirantH** [comment]
多谢开发者~
